pkg/controller/syncsetinstance/syncsetinstance_controller.go

```diff
@@ -71,19 +73,30 @@ type Applier interface {
 	ApplyRuntimeObject(obj runtime.Object, scheme *runtime.Scheme) (hiveresource.ApplyResult, error)
 }
 
-// Add creates a new SyncSet controller and adds it to the manager with default RBAC. The manager will set fields on the
+// Add creates a new SyncSetInstance controller and adds it to the manager with default RBAC. The manager will set fields on the
 // controller and start it when the manager starts.
 func Add(mgr manager.Manager) error {
-	return AddToManager(mgr, NewReconciler(mgr))
+	logger := log.WithField("controller", controllerName)
+	reapplyInterval := defaultReapplyInterval
+	var err error
+	if envReapplyInterval := os.Getenv(reapplyIntervalEnvKey); len(envReapplyInterval) > 0 {
+		reapplyInterval, err = time.ParseDuration(envReapplyInterval)
+		if err != nil {
+			log.WithError(err).Errorf("unable to parse %s: %s", reapplyIntervalEnvKey, envReapplyInterval)
```

Comment:
Log `envReappyInterval` as a field rather than in the error message.
```
log.WithError(err).WithField("reapplyInterval", envReapplyInterval).Errorf("unable to parse %s", reapplyIntervalEnvKey)
```